src/main/scala/mesosphere/marathon/api/v2/validation/AppValidation.scala

```diff
@@ -304,7 +305,7 @@ trait AppValidation {
     }
   )
 
-  def validateCanonicalAppUpdateAPI(enabledFeatures: Set[String]): Validator[AppUpdate] = forAll(
+  def validateCanonicalAppUpdateAPI(enabledFeatures: Set[String], normalizationConfig: AppNormalization.Config): Validator[AppUpdate] = forAll(
```

Comment:
sure, refactor some bits in the migration code: extract the network name logic into a separate `def` that's reused by the normalization init code